torchvision/models/shufflenetv2.py

```diff
@@ -104,6 +123,10 @@ def __init__(self, stages_repeats, stages_out_channels, num_classes=1000, invert
 
         self.maxpool = nn.MaxPool2d(kernel_size=3, stride=2, padding=1)
 
+        # Static annotations for mypy
+        self.stage2: nn.Sequential
+        self.stage3: nn.Sequential
+        self.stage4: nn.Sequential
```

Comment:
note: it would have been preferable to let `stage{}` be a `nn.Module`, as they could potentially be any type of `nn.Module`, not just `Sequential`